reg.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import, division, print_function
```

Comment:
choose a descriptive file name.

Reply:
by descriptive file name, you mean change from reg.py to something.py ?

Reply:
to like model_train.py or something. Later we will move the model to a separate module called model.py. And we will have a model_eval.py module as well.